src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -743,6 +743,13 @@ else if (authorizedSourcePattern != null
                                         transportNamespace,
                                         channelIQ.isInitiator(),
                                         channelIQ.getRTPLevelRelayType());
+
+                                if (channel instanceof VideoChannel)
+                                {
+                                    VideoChannel videoChannel = (VideoChannel)channel;
+                                    videoChannel.setReceiveSimulcastLayer(channelIQ.getReceivingSimulcastLayer());
```

Comment:
Awesome, thank you @lluczo! Yet another (very) minor thing, could you please make sure this line doesn't exceed 80 chars? Thank you very much!